client/main.go

```diff
@@ -45,6 +46,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Verify IP address
+	if utils.VerifyIP(options.Interface.IP) {
+		log.Fatalf("IP address is not correct: %s", options.Interface.IP)
+	}
+
+	// Verify gateway address
+	if utils.VerifyCIDR(options.Interface.Gateway) {
```

Comment:
This function should parse input string for CIDR address validity not `options.Interface.Gateway`
the `options.Interface.Gateway` only has IP address.

```go
if utils.VerifyCIDR(options.Interface.IP) 
```